### PR TITLE
chore: add pre-requisite step about creating issue for release to RELEASE.md

### PR DESCRIPTION
**Pull request by dgolovin**
### What does this PR do?

Adds pre-requisite step about creating issue about releasing new version for better tracking.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

N/A

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Pre-requisites
 
+- Create Enhancement Issue `Release vX.X.X` for current sprint and assign it to yourself.
```
Comment (benoitf) [inline RELEASE.md]:
```suggestion
- Create Enhancement Issue `Release vX.X.X` for current sprint, then update the label to `kind/release` and assign it to yourself.
```

Comment (benoitf) [reply RELEASE.md]: @dgolovin ?

Comment (benoitf) [reply RELEASE.md]: @dgolovin ?

**Comment by cdrage** [comment]
Hi @dgolovin if you can sign off @benoitf 's suggestion, let's get this merged in 👍

**Review by benoitf — APPROVED** [review]
Thanks @dgolovin

**Review by deboer-tim — APPROVED** [review]

**Pull request merged**